archetypes/Portfolio/History/index.tsx

```diff
@@ -162,82 +205,229 @@ const CommitRow: React.FC<
-            <TableRowCell>
-                <div>{timeString}</div>
-                <div className="text-cool-gray-500">{dateString}</div>
-            </TableRowCell>
-            <TableRowCell>
-                <div className="flex my-auto">
-                    <Logo size="lg" ticker={tokenSymbolToLogoTicker(tokenSymbol)} className="inline my-auto mr-2" />
-                    <div>
-                        <div className="flex">
+
+    const HistoryTableRow = () => {
+        if (type === COMMIT_TYPES_V2.LONG_MINT || type === COMMIT_TYPES_V2.SHORT_MINT) {
+            return (
+                <TableRow key={index} rowNumber={index}>
+                    {/*Time / Date*/}
+                    <TableRowCell>
+                        <div>{timeString}</div>
+                        <div className="text-cool-gray-500">{dateString}</div>
+                    </TableRowCell>
+                    {/*Token*/}
+                    <TableRowCell>
+                        <div className="flex my-auto">
+                            <Logo
+                                size="lg"
+                                ticker={tokenSymbolToLogoTicker(tokenOutSymbol)}
+                                className="inline my-auto mr-2"
+                            />
                             <div>
-                                {tokenSymbol.split('-')[0][0]}-{marketSymbolToAssetName[tokenSymbol.slice(3)]}
+                                <div className="flex">
+                                    <div>
+                                        {tokenOutSymbol.split('-')[0][0]}-
+                                        {marketSymbolToAssetName[tokenOutSymbol.slice(3)]}
+                                    </div>
+                                    &nbsp;
+                                    <div className={type === COMMIT_TYPES_V2.LONG_MINT ? 'green' : 'red'}>
+                                        {type === COMMIT_TYPES_V2.LONG_MINT ? 'Long' : 'Short'}
+                                    </div>
+                                </div>
+                                {tokenOutSymbol}
                             </div>
-                            &nbsp;
-                            <div className={type === 'LongMint' || type === 'LongBurn' ? 'green' : 'red'}>
-                                {type === 'LongMint' || type === 'LongBurn' ? 'Long' : 'Short'}
+                        </div>
+                    </TableRowCell>
+                    {/*Amount*/}
+                    <TableRowCell>
+                        {toApproxCurrency((price.toNumber() / 1e18) * (tokenInAmount.toNumber() / 1e18))}
+                    </TableRowCell>
+                    {/*Tokens / Price*/}
+                    <TableRowCell>
+                        <div>{(tokenInAmount.toNumber() / 1e18).toFixed(2)} tokens</div>
+                        <div className="text-cool-gray-500">
+                            at {toApproxCurrency(price.toNumber() / 1e18)} USD/token
```

Comment:
I have updated the api to return name and symbol of the pricing token, we'll need to replace any hardcoded `USD`'s with this